pkg/aws_cloudwatch_test.go

```diff
@@ -61,6 +61,120 @@ func TestSortyByTimeStamp(t *testing.T) {
 	require.Equal(t, expectedDataPoints, sortedDataPoints)
 }
 
+func Test_getFilteredMetricDatasBadData(t *testing.T) {
```

Comment:
Instead of having a separate test, could you merge this example with the test table in `Test_getFilteredMetricDatas`?

Reply:
done!